router/pkg/metric/metric_store.go

```diff
@@ -33,6 +33,8 @@ const (
 
 	unitBytes        = "bytes"
 	unitMilliseconds = "ms"
+
+	RouterConfigVersion = "router.config.version"
```

Comment:
Please take a look at the other metrics. They are stored above and naming is different. We also wanted to give the metric a more generic name to keep it flexible similar to `target_info`